templates/Deepcell_Postprocessing.ipynb

```diff
@@ -14,12 +14,16 @@
    "outputs": [],
```

Comment:
<p>@ngreenwald, correct me if I'm wrong but shouldn't <code>visualize_segmentation</code>  handle the <code>imshow</code>  logic internally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>